test/splitview/public/index.html

```diff
@@ -31,51 +31,131 @@
 	<script>
```

Comment:
Great that you added tests! I see they pass, I will not review them.

Reply:
@isidorn Everything in `test/splitview` are just visual manual tests, like a playground, you can safely ignore that. Proper unit tests are in place to make sure all this works.